src/Maker/MakeMessage.php

```diff
@@ -97,9 +100,19 @@ public function generate(InputInterface $input, ConsoleStyle $io, Generator $gen
             'Handler'
         );
 
+        $useStatements = new UseStatementGenerator([]);
+
+        if ($chosenTransport && class_exists(AsMessage::class)) {
+            $useStatements->addUseStatement(AsMessage::class);
+        }
```

Comment:
Jesse & I got in the habit of adding something like `@legacy to support < symfony/messenger 7.2`